src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/ReferenceConfidenceModelUnitTest.java

```diff
@@ -508,6 +508,10 @@ public void testRefConfidenceBasic(final int nReads, final int extension) {
         final IndependentSampleGenotypesModel genotypingModel = new IndependentSampleGenotypesModel();
         final List<Integer> expectedDPs = Collections.nCopies(data.getActiveRegion().getSpan().size(), nReads);
         final List<VariantContext> contexts = model.calculateRefConfidence(data.getRefHap(), haplotypes, data.getPaddedRefLoc(), data.getActiveRegion(), likelihoods, ploidyModel, calls, false, Collections.emptyList());
+        // Asserting that none of the reads after calculateRefConfidence have indel informativeness caching values attached.
+        for (GATKRead read : data.getActiveRegion().getReads()) {
+            Assert.assertTrue(read.getTransientAttribute(ReferenceConfidenceModel.INDEL_INFORMATIVE_BASES_CACHE_ATTRIBUTE_NAME) == null);
```

Comment:
Use `Assert.assertNull()` here.

Reply:
Also, should checking for the clearing of this attribute be its own test case, instead of piggybacking on other test cases?

Reply:
I'm worried about clearing the cache in every case so I would prefer to assert that it isn't leaving the reads cached in any case, thus I attached this to all of our tests.